src/lib/binutils/RelocateCubin.cpp

```diff
@@ -319,8 +340,13 @@ relocateLineMap
 #endif
               if (n_relocations > 0) {
                 Elf_Data *relocations_data = elf_getdata(scn, NULL);
-                applyLineMapRelocations(symbol_values, line_map,
-                  n_relocations, relocations_data);
+                if (shdr.sh_type == SHT_RELA) {
+                  applyDebugInfoRelocations(symbol_values, line_map,
+                    n_relocations, relocations_data);
+                } else {
+                  applyLineMapRelocations(symbol_values, line_map,
```

Comment:
the name of applyLineMapRelocations should also be changed to applyRELrelocations